src/pipecat/services/ai_services.py

```diff
@@ -217,6 +257,106 @@ async def request_image_frame(self, user_id: str, *, text_content: Optional[str]
             UserImageRequestFrame(user_id=user_id, context=text_content), FrameDirection.UPSTREAM
         )
 
+    async def _run_function_call(
+        self,
+        context: OpenAILLMContext,
+        tool_call_id: str,
+        function_name: str,
+        arguments: str,
+        run_llm: bool = True,
+    ):
+        if function_name in self._functions.keys():
+            entry = self._functions[function_name]
+        elif None in self._functions.keys():
```

Comment:
Just for my understanding: if a function is registered with no name (`None`), then it will be called anytime a function with an "unrecognized" name is called?